test/interpreter/function-sign.spec.ts

```diff
@@ -0,0 +1,36 @@
+import {HyperFormula} from '../../src'
+import {ErrorType} from '../../src/Cell'
+import {ErrorMessage} from '../../src/error-message'
+import {adr, detailedError} from '../testUtils'
+
+describe('Function SIGN', () => {
+  it('should not work for wrong number of arguments', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=SIGN()'],
+      ['=SIGN(1, 2)'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+    expect(engine.getCellValue(adr('A2'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+  })
+
+  it('should not work for arguments of wrong type', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=SIGN("foo")'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.VALUE, ErrorMessage.NumberCoercion))
+  })
+
+  it('should work', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=SIGN(5)'],
+      ['=SIGN(0)'],
```

Comment:
Just to be sure. `=SIGN(-0)` will be handled by us, and `-0` will be normalized before it gets to the SIGN function?

Reply:
of course